### ISSUE TITLE
Embed YouTube in Privacy friendly mode

### ISSUE THREAD

**Issue by trieloff** [adobe/helix-embed#258]
https://dri.es/how-to-remove-youtube-tracking

**Comment by chenjefferson** [comment]
Hi! First time contributor here who would like to work on this issue. Is it still open?

**Comment by trieloff** [comment]
Yes, feel free to take it on. Please note that in order to merge the PR, we need you to agree to the Adobe CLA https://opensource.adobe.com/cla.html
